tap_google_sheets/singer_transform.py

```diff
@@ -0,0 +1,403 @@
+import datetime
```

Comment:
Add file comment and mention the reason for putting singer_transform functions here.

Reply:
removed the file